Makefile

```
@@ -30,8 +30,8 @@ test-e2e-release: build
 	PASSES="release e2e" ./scripts/test.sh $(GO_TEST_FLAGS)
 
 .PHONY: test-linearizability
-test-linearizability:
-	FAILPOINTS=true PASSES="linearizability" ./scripts/test.sh $(GO_TEST_FLAGS)
+test-linearizability: gofail-enable
```

It doesn't make sense to depend on `gofail-enable`.  It should only depend on binaries built with gofail enabled. 
```suggestion
test-linearizability: 
```
```suggestion
test-linearizability:
```